scripts/azure/storage-manager.sh

```diff
@@ -0,0 +1,111 @@
+#!/bin/bash
+
+set -e
+
+readonly CREATE_ACTION="create"
+readonly DELETE_ACTION="delete"
+
+function usage() {
+    echo "Usage:" 1>&2
+    echo " $0 create" 1>&2
+    echo " $0 delete" 1>&2
+}
+
+case "$1" in
+create)
+    ACTION=$CREATE_ACTION
+    ;;
+delete)
+    ACTION=$DELETE_ACTION
+    ;;
+esac
+shift
+
+function create_resource_group() {
+
+    echo "checking if resource group exists" >&2
+    GROUP_EXISTS=$(az group list --query '[].name' | awk "/${CK8S_ENVIRONMENT_NAME}-storage-resource-group/")
+    if [ "$GROUP_EXISTS" ]; then
+        echo "resource group [${CK8S_ENVIRONMENT_NAME}-storage-resource-group] already exists" >&2
+        echo "contnue using this group ? (y/n)" >&2
+        # shellcheck disable=SC2162
+        read -n 1 cmdinput
+        case "$cmdinput" in
+        y) return ;;
+        n) exit 0 ;;
+        esac
+    else
+        echo "resource group [${CK8S_ENVIRONMENT_NAME}-storage-resource-group] does not exist, creating it now" >&2
+        az group create \
+            --name "$CK8S_ENVIRONMENT_NAME"-storage-resource-group \
+            --location swedencentral --only-show-errors
```

Comment:
Do we think that we'll use this in some other location than sweden for the forseable future? If so, then maybe make this configurable

```suggestion
            --location "${AZURE_LOCATION}" --only-show-errors
```

Reply:
I believe we will, I'll change that